Packs/CimTrak-SystemIntegrityAssurance/IncidentTypes/incidenttype-Incident_Type.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
@kigerjoel I believe this file is not needed.

Reply:
Fixed